### PR TITLE
Add Djikstra search method

### PR DESCRIPTION
**Pull request by matiuszaa**

### CODE REVIEW

File: graph.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * Function finds the shortest paths from start to all vertices in graph. 
+ * @param int vertices - number of all vertices in graph. 
+ * @param float **graph - Represented as adjacency matrix.
+ * @param int *distance - Array of the shortest paths from start.
+ * @param int *path - Array of previous vertices.  
+ * @return -1 if failed 0 if successful.
+ */
+int dijkstra_search(int vertices, float **graph, int start, int *distance, int *path); 
+
+#endif
```
Comment (Cinek28) [inline graph.h]: Add new line at the end of file

File: graph.c [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    graph[start][start] = VERTEX_USED;
+
+    /* Checking every vertex of array */
+    for (i = 0; i < vertices; ++i)
+    {
+        shortest = 0;
+
+        /* Go past through used vertices */
+        while (graph[shortest][shortest] != VERTEX_NOT_USED)
+            shortest++;
```
Comment (Cinek28) [inline graph.c]: ++shortest better here

File: graph.h (hunk 1 above)
Comment (matiuszaa) [reply graph.h]: Done.

File: graph.c (hunk 2 above)
Comment (matiuszaa) [reply graph.c]: Done.

**Pull request merged**